Saas/Publish-PerTenantExtensionApps.ps1

```diff
@@ -42,6 +42,8 @@ function Publish-PerTenantExtensionApps {
         $appFiles,
         [ValidateSet('Add','Force')]
         [string] $schemaSyncMode = 'Add',
+        [ValidateSet('Current','Next minor','Next major')]
+        [string] $schedule,

```

Comment:
Please modify the values to be the same as the supported values in schedule and then only set body.schedule if schedule is not blank.
Also we need to support not setting schedule at all - maybe allow blank and have default be blank.

Reply:
changed in the last commit